### PR TITLE
Feature/improve ghactions readme

### PR DESCRIPTION
**Pull request by luqmanbello**
# Description

- [ ] Summary of the change / bug fix.
- [ ] Link # issue, if applicable.
- [ ] Screenshot of the feature or the bug before/after fix, if applicable.
- [ ] Relevant motivation and context.
- [ ] List any dependencies that are required for this change.

# How has this been tested?

* Please describe the tests that you ran to verify your changes.
* Provide instructions so we can reproduce.
* Please also list any relevant details for your test configuration.
- [ ] Make sure affected commands still run in terminal
- [ ] Ensure the SDK still works
- [ ] Check any related reports

# Checklist:

- [ ] I have adhered to the GitFlow naming convention and my branch name is in the format of `feature/feature-name` or `hotfix/hotfix-name`.
- [ ] Update [our documentation](https://openbb-finance.github.io/OpenBBTerminal/) following [these guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/tree/main/website).  Update any user guides that are affected by the changes.
- [ ] Update our tests following [these guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/tree/main/tests).
- [ ] Make sure you are following our [CONTRIBUTING guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/blob/main/CONTRIBUTING.md).
- [ ] If a feature was added make sure to add it to the corresponding [integration test script](https://github.com/OpenBB-finance/OpenBBTerminal/tree/develop/openbb_terminal/miscellaneous/integration_tests_scripts).

# Others
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.

### CODE REVIEW

File: .github/workflows/README.md [hunk 1]
```diff
@@ -1,232 +1,665 @@
...
+### 1.1. Release Process
+
+**Purpose**  
+This process is a guide on how to build a new version of the OpenBB Installer.
+
+**Steps**
+
+*   The user will Create a branch for release from `develop` branch which follows the GitFlow naming convention i.e. `release/x.y.z`
+    
+*   The user will increment the version number in 3 files: `pyproject.toml` `setup.nsis` `feature_flags.py` and commit the changes
```
Comment (jmaslek) [inline .github/workflows/README.md:40]: There are actually quite a few files to edit.

File: .github/workflows/README.md [hunk 2]
```diff
@@ -1,232 +1,665 @@
...
+    
+*   The [Build & Release]() workflow will be triggered on `push`. This will trigger the workflows to build the installers, publish the docker image and deploy to Pypi.
+    
+*   The User merges the `release` branch to the `main` and `develop` branch once the release process is completed successfully.
```
Comment (jmaslek) [inline .github/workflows/README.md:44]: Also need to do the release drafter to generate the new release on github.

File: .github/workflows/README.md (hunk 1 above)
Comment (luqmanbello) [reply .github/workflows/README.md:40]: Can you provide the others so I can add them here?

File: .github/workflows/README.md (hunk 2 above)
Comment (luqmanbello) [reply .github/workflows/README.md:44]: Noted

File: .github/workflows/README.md (hunk 1 above)
Comment (jmaslek) [reply .github/workflows/README.md:40]: pyproject.toml
setup.nsis
build/pyinstaller/version.rc
build/docker/compose.env
system_model.py

New readmes need to be added and the old ones need the sidebar position to be bumped

**Pull request closed without merging**